scripts/setup/config.ps1

```diff
@@ -45,13 +45,38 @@ Param (
     
     [parameter()]
     [string]
-    $Destination
+    $Destination,
+
+    [parameter()]
+    [string]
+    $IISAdminOwnersGroup = "IIS Administration API Owners",
+
+    [parameter()]
+    [switch]
+    $LegacyConfigurations
 )
 
 # Name of file we place installation data in
 $INSTALL_FILE = "setup.config"
 $IISAdminSiteName = "IISAdmin"
 
+## Ensure the local gorup exists
+function EnsureGroup($group) {
+    if (!(Get-LocalGroup -Name $group -ErrorAction SilentlyContinue)) {
+        New-LocalGroup -Name $group | Out-Null
+    }
+}
+
+## ensure the member/group exists in the specified group
+function EnsureMember($group, $userOrGroup) {
```

Comment:
This belongs in [security.ps1](https://github.com/Microsoft/IIS.Administration/blob/dev/scripts/setup/security.ps1). Also are these commandlets supported on all OS out of the box. I remember we needed to do some work to see what commandlets were available at runtime for group related actions.

Reply:
Moving in the next revision